tests/pc/test_po_voq.py

```diff
@@ -58,17 +61,24 @@ def setup_teardown(duthosts, enum_rand_one_per_hwsku_frontend_hostname):
             if '.' in ip:
                 portchannel_ip = ip
 
+    for addr in config_facts['BGP_NEIGHBOR']:
+        if portchannel_ip.split('/')[0] == config_facts['BGP_NEIGHBOR'][addr]['local_addr']:
+            nbr_addr = addr
+
+
     if len(portchannel_members) == 0:
         pytest.skip("Skip test due to there is no portchannel member exists in current topology.")
 
     voq_lag.delete_lag_members_ip(duthost, asic, portchannel_members, portchannel_ip, portchannel)
+    verify_no_routes_from_nexthop(duthosts, nbr_addr)
     voq_lag.add_lag(duthost, asic, portchannel_members, portchannel_ip)
 
     yield asic, portchannel_ip, portchannel_members
 
     voq_lag.delete_lag_members_ip(duthost, asic, portchannel_members, portchannel_ip)
     # remove tmp portchannel
     voq_lag.delete_lag(duthost, asic)
+    verify_no_routes_from_nexthop(duthosts, nbr_addr)
```

Comment:
Why do we need this verification at teardown? If the assert failed. the lag members will not be added back.

Reply:
The added portchannel in the test has the same ip as the original portchannel, and thus the same eBGP peer and also the same number of routes. Thus, when deleting the new portchannel, we need to make sure that all routes are cleared as well in order to avoid the same issue of orchagent exiting when we deleted the original portchannel.

If we are not able to delete all the routes, we are in a bad state, and by adding the original portchannel back will result in failures anyways